### ISSUE TITLE
Choice frontend? Manifest? Web app?

### ISSUE THREAD

**Issue by Thatoo** [YunoHost-Apps/pleroma_ynh#75]
Hello,

What front end (fe) is available once this package installed?
Pleroma-FE and Mastodon's front end, right?
When we click on Pleroma's tile in Yunohost, which fe will come first? Can we choose? Is it possible to switch by clicking somewhere?

Is it possible to have a web app from one of the fe?

To help you looking into this issue, I have found this two links (I'm pretty sure they are related but I don't understand well what they imply, I let you judge):
https://git.pleroma.social/pleroma/mastofe/issues/35
https://git.pleroma.social/pleroma/pleroma-fe/merge_requests/458

Would it be possible to have a manifest template file coming with the installation?
Then, as the instance administrator, if we desire to have a web app for our instance, we could modify it with the help of some explanation we could find in README.md page.

Web app are very nice.
https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Advantages
and I want to offer the possibility to my family and friend to install it as simply as clicking on one button : https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Add_to_home_screen .

And I have the feeling that it's only a question of manifest file
https://developer.mozilla.org/en-US/docs/Web/Manifest

Basically, would it be possible to get our Pleroma instance as a web App?

**Comment by yalh76** [comment]
Hi @Thatoo 


Yes both frontend are available, it's just a packaging of the original pleroma.

When clicking on the Pleroma's title in Yunohost is the default FE that comes: pleroma FE

I don't know if it's possible to choose, you have to look at pleroma documentation if it's possible to switch from one default FE to another one.

When you are on PleromaFE you can switch to mastodon FE

Having one of the fe as a web app will be possible as soon as pleroma provide it. Seems not the case for now.
